cmd/supernode/app/root.go

```diff
@@ -175,11 +176,10 @@ func runSuperNode() error {
 // initLog initializes log Level and log format.
 func initLog(logger *logrus.Logger, logPath string) error {
 	logFilePath := path.Join(options.HomeDir, "logs", logPath)
```

Comment:
Make a record here.

Reply:
sorry,please in detail ,I can't understand